### ISSUE TITLE
This plugin swallows all errors and makes local debugging impossible

### ISSUE THREAD

**Issue by kylewelsby** [MatteoGabriele/vue-analytics#95]
If you are reporting a bug, please fill in below. Otherwise feel free to remove this template entirely.

### Description

[This bit of code](https://github.com/MatteoGabriele/vue-analytics/blob/master/src/lib/exception.js#L11-L16) needs to re-raise the handler like [other plugins do](https://github.com/getsentry/raven-js/blob/0fbdf951067e245bb605f530a4cb7f4f143f5ea7/plugins/vue.js#L24-L44)

### Expected behavior

Allow standard error handing to continue to be omitted to the browser to assist development. 

### Actual behavior

All errors are being swallowed.

### Environment

Development

**Comment by MatteoGabriele** [comment]
hi @kylewelsby thanks for the feedback

could you please provide a public repository with a simple application done with `vue-cli` and this issue proved with and without the plugin installed?

Thanks a lot for your time 

cheers

**Comment by kylewelsby** [comment]
Hey Matteo,
It will be quite some time before I get to commit some preview code regarding this.  About April time when I'm back at a computer, taking some time off.

**Comment by MatteoGabriele** [comment]
No worries, enjoy your time off and reopen this issue as soon as you have a reproduction

Cheers :)

**Comment by tomreid76** [comment]
FWIW, I am seeing this issue as well. Took me a while to track down, but the `5.8.0` release is where it began. It is swallowing some, but not all, errors. `5.7.1` is fine.

Missing data or computed properties that are relied on in my views will still surface errors. Errors thrown in imperative code not directly relied on in the view (for instance reference errors) will fail silently.

**Comment by MatteoGabriele** [comment]
I will revert to the previous error handling: just listening to the window error exceptions

**Comment by fiws** [comment]
.... we just lost half a day because of this :\

**Comment by MatteoGabriele** [comment]
@fiws I'm sorry for that!

I'm going to fix it tomorrow. 

When nobody can prove the issue is kinda hard for me to fix it, but I will just rollback to the previous behavior.

**Comment by fiws** [comment]
@MatteoGabriele it's ok. shit happens.

Just throw an error in the mounted hook inside your main App.vue to reproduce this.

**Comment by fiws** [comment]
Demo: https://codesandbox.io/s/q82zv7xwj

**Comment by MatteoGabriele** [comment]
because Vue is handling errors itself, I can't use `window.onerror` or eventListeners because they are swallowed already by Vue

so this could be the new solution for that: it will track errors but they will be silent errors, because of `Vue.config.errorHandler`

it is still possible to add a `Vue.config.errorHandler` **before** installing the plugin and it will be fired as well

```js
  if (!config.autoTracking.exception) {
    return
  }

  const oldErrorHandler = Vue.config.errorHandler

  Vue.config.errorHandler = (error, vm, info) => {
    exception(error.message)

    if (typeof oldErrorHandler === 'function') {
      oldErrorHandler.call(this, error, vm, info)
    }
  }
```

**Comment by MatteoGabriele** [comment]
Bug fixed with 5.9.1
thanks all for your feedback and sorry again for the struggle
